### PR TITLE
Merge dev in esdt relayed arwen

### PR DESCRIPTION
**Pull request by miiu96**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -30,6 +30,7 @@ github.com/ElrondNetwork/elrond-go-logger v1.0.2/go.mod h1:e5D+c97lKUfFdAzFX7rrI
 github.com/ElrondNetwork/elrond-go-logger v1.0.4 h1:i5Yu4qyjTZDwvBY/ykbNpp2SP9jxwk/QTivRwSZSTAQ=
 github.com/ElrondNetwork/elrond-go-logger v1.0.4/go.mod h1:e5D+c97lKUfFdAzFX7rrI2Igl/z4Y0RkKYKWyzprTGk=
 github.com/ElrondNetwork/elrond-vm-common v0.1.23/go.mod h1:ZakxPST/Wt8umnRtA9gobcy3Dw2bywxwkC54P5VhO9g=
+github.com/ElrondNetwork/elrond-vm-common v0.3.0/go.mod h1:ZakxPST/Wt8umnRtA9gobcy3Dw2bywxwkC54P5VhO9g=
```
Comment (iulianpascalau) [inline go.sum:33]: go mod tidy?

File: go.mod [hunk 2]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/ElrondNetwork/arwen-wasm-vm v0.5.0
 	github.com/ElrondNetwork/concurrent-map v0.1.3
 	github.com/ElrondNetwork/elrond-go-logger v1.0.4
-	github.com/ElrondNetwork/elrond-vm-common v0.3.0
+	github.com/ElrondNetwork/elrond-vm-common v0.3.3
```
Comment (sasurobert) [inline go.mod:9]: there should be no reference to elrond-vm-common

Comment (iulianpascalau) [inline go.mod:9]: do we still need elrond-vm-common?

Comment (sasurobert) [reply go.mod:9]: have to be change to core/vmcommon

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: go.sum (hunk 1 above)
Comment (miiu96) [reply go.sum:33]: done.

File: go.mod (hunk 2 above)
Comment (miiu96) [reply go.mod:9]: was removed after go mod tidy

Comment (miiu96) [reply go.mod:9]: was removed after go mod tidy

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**